src/RequestHandler/BaseHandlerTrait.php

```diff
@@ -2,27 +2,39 @@
 
 namespace SilverStripe\MFA\RequestHandler;
 
-use SilverStripe\Control\HTTPResponse;
 use SilverStripe\Core\Injector\Injector;
 use SilverStripe\MFA\Service\MethodRegistry;
-use SilverStripe\MFA\Service\RegisteredMethodManager;
 use SilverStripe\MFA\Store\StoreInterface;
 use SilverStripe\Security\Member;
 use SilverStripe\View\Requirements;
 
 trait BaseHandlerTrait
 {
+    /**
+     * A "session store" object that helps contain MFA specific session detail
+     *
+     * @var StoreInterface
+     */
+    protected $store;
+
     /**
      * Perform the necessary "Requirements" calls to ensure client side scripts are available in the response
+     *
+     * @param bool $frontEndRequirements Indicates dependencies usually provided by admin should also be required
      */
-    protected function applyRequirements(): void
+    protected function applyRequirements(bool $frontEndRequirements = true): void
```

Comment:
Do you want to make this two separate methods?

Reply:
🤔 The crux of this method is looping the methods to provide their specific requirements. I think it's _just_ not worth splitting up. I can be swayed if you disagree though.